debian/control

```diff
@@ -0,0 +1,31 @@
+Source: crs-scripts
+Priority: optional
+Maintainer: derchris <derchris@me.com>
+Build-Depends: debhelper (>= 10),
+               dh-autoreconf,
+               docbook-xsl
+Standards-Version: 4.2.1
+Section: libs
+Homepage: https://github.com/crs-tools/crs-scripts
+
+Package: crs-scripts
+Architecture: any
+Depends: ${misc:Depends}, 
+         ${shlibs:Depends},
+         libconfig-inifiles-perl,
+         libfile-which-perl,
+         libipc-run3-perl,
+         libproc-processtable-perl,
+         libxml-rpc-fast-perl,
+         libxml-simple-perl,
+         libmath-round-perl,
+         libdatetime-perl,
+         libboolean-perl,
+         libwww-curl-perl,
+         libjson-perl,
+         fuse-ts,
+         fuse,
+         cifs-utils,
+         ntfs-3g
+Description: CRS Worker scripts
```

Comment:
Please also add Recommends: ca-certificates as the need for the separate ssl directory is actually blown by usage of Lets Encrypt.